### PR TITLE
You can no longer drop survival capsules on top of dense objects

### LINKED ISSUE #16731: All simple animals ranged cooldowns are too short
Each simple animal had a cooldown, in life ticks. These were converted to cooldowns in deciseconds, at a ratio of 10 deciseconds/1 second per life tick . All simple animals can fire too fast now, resulting in stuff like goliaths chainstunning people.

I'm not sure what the exact life tick/second ratio is, but I'm guessing it's closer to 1.5

### LINKED ISSUE #16747: Lavaland survival pod gps computer cannot have its tag changed
The set tag button does not change the displayed tag when ok is hit after a new name is entered in the popup.
@KorPhaeron

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
rscdel: You can no longer drop survival capsules on top of dense objects, such as the windows in the emergency shuttle. You can still drop them on non-dense objects and mobs, however.
/:cl:

Fixes #16731
Fixes #16747
Mining mobs fire far less often.
Fixes a bug where the threshhold turf had no air.
Fixes a bug where the floors had the wrong icons.
The warning messages for capsule activation have span classes.
Minerals no longer have rock edging on the border.
Iron and bananium now fade out properly.
The storage in the pod now provides some innate light, making the pod easier to find.

### CODE REVIEW

**Comment by coiax** [comment]
wait, so being near a bluespace capsule when it expands doesn't harm you? Why are we standing back?

**Comment by ChangelingRain** [comment]
sound of IDUNNO

File: code/modules/mob/living/simple_animal/hostile/mining_mobs.dm [hunk 1]
```diff
@@ -329,7 +329,7 @@
 	mouse_opacity = 2
 	move_to_delay = 40
 	ranged = 1
-	ranged_cooldown_time = 80
+	ranged_cooldown_time = 140
```
Comment (KorPhaeron) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs.dm]: might be a bit too much, though I frankly have no clue what the correct value is anymore.

Goliaths right now are definitely scary, but they also encourage team work, so ??

**Comment by MMMiracles** [comment]
gotta sneak in that one mining mob nerf within abunch of changes amiright?

File: code/modules/mining/mine_items.dm [hunk 2]
```diff
@@ -284,24 +288,20 @@
...
 
 	var/turf/threshhold = locate(start_turf.x, start_turf.y-2, start_turf.z)
-	threshhold.ChangeTurf(/turf/open/floor/plasteel/pod)
-	threshhold.blocks_air = 1 //So the air doesn't leak out
-	threshhold.initial_gas_mix = "o2=21;n2=82;TEMP=293.15"
+	threshhold.ChangeTurf(/turf/open/floor/pod)
+	var/turf/open/floor/pod/doorturf = threshhold
+	doorturf.blocks_air = 1 //So the air doesn't leak out
+	doorturf.initial_gas_mix = "o2=21;n2=82;TEMP=293.15"
+	doorturf.air.copy_from_turf(doorturf)
```
Comment (duncathan) [inline code/modules/mining/mine_items.dm]: replace these lines with
`doorturf.air.parse_gas_string("o2=21;n2=82;TEMP=293.15")`
it's my fault for being dumb, not yours

File: code/modules/mining/mine_items.dm [hunk 3]
```diff
@@ -284,24 +288,20 @@
...
 	var/list/turfs = room["floors"]
 	for(var/turf/open/floor/A in turfs)
-		SSair.remove_from_active(A)
-		A.oxygen = 21
-		A.temperature = 293.15
-		A.nitrogen = 82
-		A.carbon_dioxide = 0
-		A.toxins = 0
+		A.initial_gas_mix = "o2=21;n2=82;TEMP=293.15"
 		A.air.copy_from_turf(A)
```
Comment (duncathan) [inline code/modules/mining/mine_items.dm]: replace these two with
`A.air.parse_gas_string("o2=21;n2=82;TEMP=293.15")`
again, my fault

**Comment by ChangelingRain** [comment]
Adjusted mining mob cooldowns to a 50% increase; that should be high enough that goliaths stop chainstunning people, probably.

**Comment by coiax** [comment]
S T E A L T H N E R F

**Comment by KorPhaeron** [comment]
Merging this a few hours early because goliaths are chainstunning people to death

**Pull request merged**